src/event-handler/receive.ts

```diff
@@ -1,15 +1,32 @@
-"use strict";
+import { wrapErrorHandler } from "./wrap-error-handler";
+import { WebhookEvent, EventState } from "../types";
+
+function getHooks(
+  state: EventState,
+  eventPayloadAction: string,
+  eventName: string | string[]
```

Comment:
this too I thing we can use AllEventTypes

Reply:
> this too I thing we can use AllEventTypes

Same here. You are totally right